src/cljs/main/broadfcui/page/workspace/notebooks/notebooks_table.cljs

```diff
@@ -333,24 +333,24 @@
            notebooks-to-localize (filter (every-pred (comp (partial = (:clusterName cluster)) :cluster-name)
                                                      (comp false? :localized?))
                                          notebooks)]
-       (endpoints/localize-notebook (:googleProject cluster) (:clusterName cluster)
-                                    (reduce merge (conj (map (partial this :-localize-entry) notebooks-to-localize) (this :-delocalize-json)))
-                                    (fn [{:keys [success? get-parsed-response]}]
-                                      (if success?
-                                        (let [updated-notebooks (map (fn [n]
-                                                                       (if (= (:cluster-name n) (:clusterName cluster))
-                                                                         (merge n {:localized? true})
-                                                                         n))
-                                                                     notebooks)]
-                                          ; flip :localized? to true so we don't re-localize notebooks to this cluster
-                                          (swap! state assoc :server-response
-                                                 {:notebooks (map (fn [n]
-                                                                    (if (= (:cluster-name n) (:clusterName cluster))
-                                                                      (merge n {:localized? true})
-                                                                      n))
-                                                                  notebooks)})
-                                          (this :-persist-notebook-cluster-association updated-notebooks))
-                                        (js/setTimeout #(this :-localize-notebooks cluster) 10000))))))
```

Comment:
No longer retrying after a timeout on errors. In practice this has led to infinite loops.